### ISSUE TITLE
Is your code correct !?

### ISSUE THREAD

**Issue by GoustiFruit** [KenanY/glicko2-lite#23]
Hi,
I just implemented a version inspired by your code, my first test gives me the same result as in your example (glicko2(1500, 350, 0.06, [[2000, 70, 0]]) -> "rating: 1467.5878493169462, rd: 318.6617548537152,  vol: 0.059999457650202655) but if I change the score between players A and B, the result is out of whack: glicko2(1500, 350, 0.06, [[2000, 70, 1]]) -> "rating: 4444,27007921906, rd: 745,314935348793, vol: 17,3667051255026" !
Is that supposed to be like that !? or is there a problem in your code !?
Thank you for spending some time to give me an answer :-)

**Comment by KenanY** [comment]
> glicko2(1500, 350, 0.06, [[2000, 70, 1]]) -> "rating: 4444,27007921906, rd: 745,314935348793, vol: 17,3667051255026"

This is with _your_ code? Just tried with glicko2-lite and I get:

```
glicko2(1500, 350, 0.06, [[2000, 70, 1]])
{ rating: 2038.2176891691993,
  rd: 318.66178535036266,
  vol: 0.06000728185418398 }
```

**Comment by GoustiFruit** [comment]
HI,

Sorry for letting you so long without any answer :-\
Yes, it's with my code, I'm not using JS, so I tried to convert your work
to my needs - using Object Pascal (Delphi) for that goal.
I checked multiple times my code in detail and can't find anything for now;
I'm now (slowly) converting another source (in Python) so maybe I'll be
able to understand what's wrong with my code, as I find the Python code
easier to understand.
I will tell you whenever I finish it, if I get another result or if there
is something wrong somewhere :-p

Regards,
Gousti.

Le mer. 3 juil. 2019 à 02:14, Kenan Yildirim <notifications@github.com> a
écrit :
